src/components/AuthLogin/EmailLogin.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from "react";
+
+import { cn } from "@/lib/utils";
+import { Button } from "../Button";
+import { TextInput } from "../TextInput";
+
+const EmailLogin = ({ className }: { className: string }) => {
+  const [error, setError] = useState<boolean>(false);
+  const [email, setEmail] = useState<string>("");
+  const [open, setOpen] = useState<boolean>(false);
+  const [password, setPassword] = useState<string>("");
+
+  const HandleSubmit = (e) => {
+    if (!open) {
+      setOpen(true);
+    }
+    return;
+  };
+
+  const openEmailLogin = () => {
+    if (!open) {
+      setOpen(true);
+    }
+  };
+
```

Comment:
```suggestion
  if (open)
    return (
      <Button className="h-12" onClick={openEmailLogin} key="open email">
        Login with Email
      </Button>
    );
```

You could use another return statement instead, makes the code much cleaner